cpp/ql/test/query-tests/Critical/MissingCheckScanf/.clang-format

```diff
@@ -0,0 +1 @@
+DisableFormat: true
```

Comment:
I think you committed this file by mistake?

Reply:
I didn't. But I guess I should keep this kind of file to myself? What do you think of .gitignore-ing it?

Reply:
We could add it to `.gitignore`, but that requires some care, as there are places where we have added `.clang-format` files on purpose. Probably best to leave this out of this PR and discuss separately. In general, I don't think we want to litter the `cpp/ql/test` directory with `.clang-format` files. 

I'm not sure what triggered you to add the `.clang-format` file, but if it was some post-commit hook, it might also be an indication that the hook needs some fine tuning.

Personally, I would also leave the change you made to the `.gitignore` file out of this PR, as that affects people outside our team. However, if you think it's a non-controversial change, then feel free to leave it as is.